### ISSUE TITLE
Range filter using 0 as parameter does not work

### ISSUE THREAD

**Issue by ramiroaznar** [CartoDB/carto.js#2201]
### Context

* Reported by SB/19254690
* `0` is not supported as a valid parameter in CARTO.js range filter.

### Steps to Reproduce

* Create a map with a range filter `gte: 0`.

```html
<!DOCTYPE html>
<html>
  <head>
    <title>Range Filter | CARTO</title>
    <meta name="viewport" content="initial-scale=1.0">
    <meta charset="utf-8">
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,600,700|Open+Sans:300,400,600" rel="stylesheet">
    <!-- Include Leaflet -->
    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
    <link href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css" rel="stylesheet">
    <!-- Include CARTO.js -->
    <script src="https://libs.cartocdn.com/carto.js/v4.1.5/carto.min.js"></script>
    <link href="https://carto.com/developers/carto-js/examples/maps/public/style.css" rel="stylesheet">
  </head>
  <body>
    <div id="map"></div>

    <script>

      const map = L.map('map').setView([40.42252398976147, -3.659729361534119], 12);
      map.scrollWheelZoom.disable();

      L.tileLayer('https://{s}.basemaps.cartocdn.com/rastertiles/voyager_nolabels/{z}/{x}/{y}.png', {
        maxZoom: 18
      }).addTo(map);

      const client = new carto.Client({
        apiKey: 'default_public',
        username: 'ramirocartodb'
      });

      const priceFilter = new carto.filter.Range('price', { gte: 1 });

      const source = new carto.source.SQL('SELECT * FROM listings');
      source.addFilter(priceFilter);

      const style = new carto.style.CartoCSS(`
        #layer {
          marker-width: 7;
          marker-fill: #EE4D5A;
          marker-line-color: #FFFFFF;
          marker-fill: ramp([price], (#ffc6c4, #ee919b, #cc607d, #9e3963, #672044), quantiles);
        }
      `);
      const layer = new carto.layer.Layer(source, style);

      client.addLayer(layer);
      client.getLeafletLayer().addTo(map);

    </script>
  </body>
</html>
```

### Current Result

* An error is shown in the console: `Invalid parameter type for 'gte'. Please check filters documentation.`

![image](https://user-images.githubusercontent.com/5215798/46818365-5415f300-cd81-11e8-99bb-c411fe987b47.png)

### Expected result

* `0` should be accepted as parameter.

### Browser and version

* Google Chrome Versión 67.0.3396.87 (Build oficial) (64 bits)

**Comment by Jesus89** [comment]
This happens also in `lt`, `lte`,  and `gt`.

**Comment by Jesus89** [comment]
This should be solved in `v4.1.6`.

```html
<!DOCTYPE html>
<html>
  <head>
    <title>Range Filter | CARTO</title>
    <meta name="viewport" content="initial-scale=1.0">
    <meta charset="utf-8">
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,600,700|Open+Sans:300,400,600" rel="stylesheet">
    <!-- Include Leaflet -->
    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
    <link href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css" rel="stylesheet">
    <!-- Include CARTO.js -->
    <script src="https://libs.cartocdn.com/carto.js/v4.1.6/carto.min.js"></script>
    <link href="https://carto.com/developers/carto-js/examples/maps/public/style.css" rel="stylesheet">
  </head>
  <body>
    <div id="map"></div>

    <script>

      const map = L.map('map').setView([40.42252398976147, -3.659729361534119], 12);
      map.scrollWheelZoom.disable();

      L.tileLayer('https://{s}.basemaps.cartocdn.com/rastertiles/voyager_nolabels/{z}/{x}/{y}.png', {
        maxZoom: 18
      }).addTo(map);

      const client = new carto.Client({
        apiKey: 'default_public',
        username: 'ramirocartodb'
      });

      const priceFilter = new carto.filter.Range('price', { gte: 0 });

      const source = new carto.source.SQL('SELECT * FROM listings');
      source.addFilter(priceFilter);

      const style = new carto.style.CartoCSS(`
        #layer {
          marker-width: 7;
          marker-fill: #EE4D5A;
          marker-line-color: #FFFFFF;
          marker-fill: ramp([price], (#ffc6c4, #ee919b, #cc607d, #9e3963, #672044), quantiles);
        }
      `);
      const layer = new carto.layer.Layer(source, style);

      client.addLayer(layer);
      client.getLeafletLayer().addTo(map);

    </script>
  </body>
</html>
```
